### PR TITLE
Add OpenCL/vendors, change lib dir

### PR DESCRIPTION
**Pull request by tpkessler**
* Copy amdocl64.icd to /etc/OpenCL/vendors. Otherwise clinfo doesn't
  work.

* Change ld.so.conf to /opt/rocm/lib/x86_64 so that the runtime can
  detect libamdocl64.so

### CODE REVIEW

File: rocm-opencl-runtime/PKGBUILD [hunk 1]
```diff
@@ -99,7 +99,10 @@ package_rocm-opencl-runtime() {
     DESTDIR="$pkgdir/" make -C "$srcdir/rocm-opencl-runtime/build" install
 
     mkdir -p "$pkgdir/etc/ld.so.conf.d"
-    echo '/opt/rocm/lib' > "$pkgdir/etc/ld.so.conf.d/rocm-opencl.conf"
+    echo '/opt/rocm/lib/x86_64' > "$pkgdir/etc/ld.so.conf.d/rocm-opencl.conf"
```
Comment (ranisalt) [inline rocm-opencl-runtime/PKGBUILD:102]: Please use ```
install -Dm644 /dev/stdin "$pkgdir/etc/ld.so.conf.d/$pkgname.conf <<-EOF
  /opt/rocm/lib/x86_64
EOF
```

File: rocm-opencl-runtime/PKGBUILD [hunk 2]
```diff
@@ -99,7 +99,10 @@ package_rocm-opencl-runtime() {
...
+
+    mkdir -p "$pkgdir/etc/OpenCL/vendors"
+    install -Dm644 "$srcdir/rocm-opencl-runtime/api/opencl/config/amdocl64.icd" "$pkgdir/etc/OpenCL/vendors/amdocl64.icd"
```
[the remark this answers is not in the document]
Comment (tpkessler) [reply rocm-opencl-runtime/PKGBUILD:105]: You're right. That is what `-D` is for.

File: rocm-opencl-runtime/PKGBUILD (hunk 1 above)
Comment (tpkessler) [reply rocm-opencl-runtime/PKGBUILD:102]: Thanks. I've changed it.

**Comment by tpkessler** [comment]
First of all, thank you for your quick review.

> Note that we are changing the way opencl-runtime and device-libs are build, splitting the package. Are you okay if we incorporate your changes in another PR?

Sure. Are you referring to #73?

**Comment by tpkessler** [comment]
I messed up my last commit. I close this PR and open a new one

**Comment by rigred** [comment]
@farnoy Please aurpublish rocminfo update
If possible add me as co-maint
Thanks

**Pull request closed without merging**